From 60df641f8592ad054347971dc3dff4e08d938a2f Mon Sep 17 00:00:00 2001
From: Tobias Mueller <muelli@cryptobitch.de>
Date: Mon, 28 Sep 2020 21:48:15 +0200
Subject: [PATCH] disabled updater

---
 main.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/main.js b/main.js
index 8d1d5bd..8e0a23e 100644
--- a/main.js
+++ b/main.js
@@ -1,7 +1,7 @@
 const electron = require('electron')
 // Module to control application life.
 const { app, BrowserWindow } = electron
-const updater = require('./updater')
+//const updater = require('./updater')
 
 // Keep window state
 const windowStateKeeper = require('electron-window-state')
@@ -136,7 +136,7 @@ function createWindow() {
     require('electron').shell.openExternal(url);
   });
 
-  setTimeout(updater.check, 2000);
+  //setTimeout(updater.check, 2000);
 }
 
 if (!gotTheLock) {
-- 
2.25.1

